src/app/core/app-config/app-config.ts

```diff
@@ -34,46 +35,55 @@ import { IAppConfig } from "./app-config.model";
  * // no need for dependency injection here
  */
 export class AppConfig {
```

Comment:
possibly take the chance to rename this to get away from the very similar, confusing "AppConfig" (here) and "ConfigService" (config from the db)?

I just can't think of anything much clearer ... maybe "CoreConfig" or "PlatformConfig" (in contrast to the ConfigService that contains use case details). If we can't come up with something better, we can also leave it as is, though.

Reply:
also see my comment below on environment.ts somewhat related to this